src/test/resources/application-test.yml

```
@@ -63,4 +63,8 @@ spring:
 
 jwt:
   secret: qwdqwei83qwiu99wq8f9qw98asf8p219qnf43yj90j32
-  valid-time: 7200000
\ No newline at end of file
+  valid-time: 7200000
+
+scheduler:
+  season:
+    rollover-cron: "* * * * * *"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)